docs/pages/tutorial/configuration.mdx

```diff
@@ -79,6 +79,20 @@ Setting the `"splash.image"` property with a valid file path in the [**app.json*
   className="max-w-[250px]"
 />
 
+Add the following `splash` object in **app.json** to configure the splash screen:
```

Comment:
Due to upcoming config plugin changes coming for `expo-splash-screen`, we need to update the tutorial chapter to add the `splash` object manually in the app config.

cc: @Simek